tests/test_vjp.py

```diff
@@ -182,15 +71,70 @@ def test_provide_starting_state(self, tol, dev):
 
         dy = np.array([1.0])
 
-        fn1 = dev.vjp(tape, dy)
+        fn1 = dev.vjp(tape.measurements, dy)
         vjp1 = fn1(tape)
 
         qml.execute([tape], dev, None)
-        fn2 = dev.vjp(tape, dy, starting_state=dev._pre_rotated_state)
+        fn2 = dev.vjp(tape.measurements, dy, starting_state=dev._pre_rotated_state)
         vjp2 = fn2(tape)
 
         assert np.allclose(vjp1, vjp2, atol=tol, rtol=0)
 
+    def test_multiple_measurements(self, tol, dev):
+        """Tests provides correct answer when provided starting state."""
```

Comment:
Maybe this sentence could use some review too.

Reply:
Yes, I also updated this one.